src/types/api/examples.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  QueryPageResult,
+  QueryPageResultWithTotal,
+  SingleFormResult,
+} from '../../app/modules/examples/examples.types'
+
+// pageNo: The page to render
+// searchTerm: The term to search on
+// agency: The agency to search on - this can be all agencies or the user's agency
+// shouldGetTotalNumResults: Whether to return all the results or not
+export type ExampleFormsQueryDto = {
+  pageNo: number
+  searchTerm?: string
+  agency?: string
+  shouldGetTotalNumResults?: boolean
+}
```

Comment:
is this really a DTO if it's only used internally by the frontend?

Reply:
used in `src/app/modules/examples/examples.controller.ts` as part of be to verify request's body's shape